documents/motions/06.2019.1 - Regional Organizations.md

```diff
@@ -25,10 +25,11 @@ The WCA shall acknowledge Regional Organizations as Partners to support the Obje
    2. The application for Regional Organizations must include all of the following information:
       1. Name of the Regional Organization
       2. Official address and contact details
-      3. A copy of its current bylaws and, if applicable, regulations, which are in compliance with the Bylaws and Regulations of the WCA
-      4. A list of its principal directors and officers
-      5. A clear description of the area that is covered by the Regional Organization
-      6. A report on past and current WCA related activities.
+      3. Official website
```

Comment:
```suggestion
      3. Official website and logo
```
I think it would be good to require all regional organisations to have a logo - especially given how they are currently displayed on the WCA website. Only Speedcubing New Zealand does not meet this requirement currently.

Reply:
I agree with you, but part of me thinks it's silly to deny a regional organization just because they have not designed a logo. I could go either way on this suggestion.

Reply:
I'm totally fine with adding this, although I would probably list it as a separate point.

Reply:
Okay, I can live with that.

Reply:
I am also OK with listing as a separate point. If they don't have a logo design, they can just text as a picture, so it's very easy to come up with something.

Reply:
I have already included this in a `Staff review` commit that I'll be pushing live later.